templates/default/cut_tiles.sh.erb

```diff
@@ -13,6 +13,13 @@ trap 'rm $LOCK_FILE' EXIT 1 2 3
 export PATH=$PATH:/usr/local/bin
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
 
+tile_dir=$(jq -r '.mjolnir.hierarchy.tile_dir' <%= node[:valhalla][:config] %>)
+
+#remove the hierarchy dirs.  if we crashed they are left around.
+rm -rf ${tile_dir}/0
```

Comment:
yeah was just going to say that....new script has been running on dev for few days now.